### PR TITLE
chore(secret-manager): add checksum-usage to snippets

### PR DESCRIPTION
**Pull request by gwhitehawk**
Adds checksum-usage snippets to SecretManager samples.

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [x] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [x] API's need to be enabled to test (tell us)
- [x] Environment Variables need to be set (ask us to set them)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [x] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: secretmanager/src/main/java/secretmanager/AccessSecretVersion.java [hunk 1]
```diff
@@ -45,6 +47,15 @@ public static void accessSecretVersion(String projectId, String secretId, String
       // Access the secret version.
       AccessSecretVersionResponse response = client.accessSecretVersion(secretVersionName);
 
+      // Verify checksum.
+      byte[] data = response.getPayload().getData().toByteArray();
+      Checksum checksum = new CRC32C();
+      checksum.update(data, 0, data.length);
```
Comment (kurtisvg) [inline secretmanager/src/main/java/secretmanager/AccessSecretVersion.java:53]: nit: consider making this a single line (same with the other example)

**Review by kurtisvg — APPROVED** [review]

**Comment by gwhitehawk** [comment]
I have a question regarding java.util.zip.Crc32c usage - this lib only exists since Java 9, so the code doesn't compile for Java 8. Google has a lib (https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/files/Crc32c) which can be used but it's deprecated (presumably because now an SE lib exists). Is Java8-compatibility necessary in these samples? I wouldn't want to advise users to use a deprecated library - if they still use java8, they could figure out how to replace the standard checksum lib usage with a custom one.

File: secretmanager/src/main/java/secretmanager/AccessSecretVersion.java (hunk 1 above)
Comment (kurtisvg) [reply secretmanager/src/main/java/secretmanager/AccessSecretVersion.java:53]: Sorry, I must not have realized that the checksum was a void return type - this is totally fine.

**Comment by kurtisvg** [comment]
@gwhitehawk - You have 2 options:

1. Changed the `pom.xml` compileTarget property to "11" (which will cause the j8 tests to skip it) and just include a comment to the library to use if they are <=j8
2. Use the older library and leave a comment that says for J9+ they can use the stdlib

**Pull request merged**